frontend/src/scenes/saved-insights/SavedInsights.tsx

```diff
@@ -395,10 +396,10 @@ export function SavedInsights(): JSX.Element {
                                     ))}
                                 </Select>
                             </Col>
-                            <Col>
+                            <div style={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
`<div className='flex-center'>`

Always look in `global.scss` if there's a little helper that you can use instead

Reply:
Why are we not using something like tailwind to make this a bit saner? :sweat_smile:

Reply:
One of the upcoming discussions I want to have is about whether or not to push for a tailwind (or similar) inclusion in the future. A lot of these utility classes pre-existed and I've slowly added a few more to cover my favourite ones from Tailwind. Definitely soon we should look at this properly though 👍

Reply:
My first reaction is that tailwind is huge, contains a lot of stuff we'll never use and its tree shaking to bring that down adds a lot of complexity to the build process (and the 3+ different bundlers we'll be using) that it's not worth it, at least now. Having our own limited set of utility classes seems like a better choice, at least now